python/lsst/ts/cbp/component.py

```diff
@@ -192,7 +237,10 @@ async def move_azimuth(self, position: float):
         if position < -45 or position > 45:
             raise ValueError("New azimuth value exceeds Azimuth limit.")
         else:
-            await self.send_command(f"new_az={position}")
+            if self.azimuth_target != position:
```

Comment:
I fear this test may not be sufficient. What if the target matches but the actuator didn't actually arrive at the target? Surely the user should be able to try the move again?

If so, I see two reasonable options:
* Eliminate the test. If moving is simple and safe then I suggest that.
* If moves are expensive or risky, then test if the target matches AND the current position is not close enough to that position. I suspect for that test to be robust you will have to explicitly ask for the current position instead of relying on the telemetry loop to have done that (sufficiently recently) for you.

This comment applies to all actuator move methods.

Reply:
Eliminated the test.